plugins/hydra_ray_launcher/example/conf/ray_mode/aws.yaml

```diff
@@ -0,0 +1,24 @@
+# @package _global_
```

Comment:
Why do we need ray_mode?
Having it in the example will make people think it's something they should copy, and I am not convinced it's a good idea.

Can the things here be in the defaults of the plugin raw configuration?